gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagStateStore.java

```diff
@@ -47,28 +47,42 @@ public interface DagStateStore {
    */
   void cleanUp(Dag<JobExecutionPlan> dag) throws IOException;
 
+  default boolean cleanUp(DagManager.DagId dagId) throws IOException {
+    cleanUp(dagId.toString());
+    return true;
+  }
+
   /**
    * Delete the {@link Dag} from the backing store, typically upon completion of execution.
    * @param dagId The ID of the dag to clean up.
    */
+  // todo - change it to return boolean
+  @Deprecated
```

Comment:
keeping them because DagManager uses it

Reply:
yes, plus it's unfortunately too late to change the API to return a boolean